Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM ruby:alpine
+WORKDIR /code
+COPY . /code
```

Comment:
It could be a good thing to add dependencies first, install them and then add the rest to leverage docker layers caching. But we can deal with that later, when setting up CI deployment.